package.json

```diff
@@ -11,14 +11,16 @@
   "homepage": "https://github.com/nodeca/pako",
   "contributors": [
     "Andrei Tuputcyn (https://github.com/andr83)",
-    "Vitaly Puzrin (https://github.com/puzrin)"
+    "Vitaly Puzrin (https://github.com/puzrin)",
```

Comment:
Will it be ok, if i drop this field entirely? I suspect, it was done for another things, and now it's just a one more point of maintenance pain. If we start add all participators here, we need to add everyone who did PR into paco. But this page is much more informative and useful https://github.com/nodeca/pako/graphs/contributors